### PR TITLE
Create scope that contains all roles except role[s]

### PR DESCRIPTION
**Pull request by whfamily2006**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +212,14 @@ $users = User::role('writer')->get(); // Only returns users with the role 'write
 
 The scope can accept a string, a `\Spatie\Permission\Models\Role` object or an `\Illuminate\Support\Collection` object.
 
+The `HasRoles` also adds a scope to your models to scope the query to except roles:
```
Comment (drbyte) [inline README.md]: 'to except roles' might be better stated as "to exclude roles".

File: README.md [hunk 2]
```diff
@@ -212,6 +212,14 @@ $users = User::role('writer')->get(); // Only returns users with the role 'write
...
+
+```php
+$users = User::exceptRole('writer')->get(); // Only returns users which not having the role 'writer'.
+```
+
+The scope can accept a array.
```
Comment (drbyte) [inline README.md]: "a array" should be "an array"

File: README.md [hunk 3]
```diff
@@ -212,6 +212,14 @@ $users = User::role('writer')->get(); // Only returns users with the role 'write
 
 The scope can accept a string, a `\Spatie\Permission\Models\Role` object or an `\Illuminate\Support\Collection` object.
 
+The `HasRoles` also adds a scope to your models to scope the query to except roles:
+
+```php
+$users = User::exceptRole('writer')->get(); // Only returns users which not having the role 'writer'.
```
Comment (drbyte) [inline README.md]: "which not having the role" should be "which do not have the role"

**Comment by freekmurze** [comment]
Thank you for this, but I'm going to pass. 

I feel that the majority of users don't need this scope and those who do can easily add it into their own project.

**Comment by whfamily2006** [comment]
@freekmurze should I change according to @drbyte or not?

**Comment by freekmurze** [comment]
I'm not going to pull in your PR, but sure you can follow @drbyte 's suggestion in your own project.

**Pull request closed without merging**